src/main/java/io/electrum/vas/model/Originator.java

```diff
@@ -106,6 +103,27 @@ public void setOperatorId(String operatorId) {
       this.operatorId = operatorId;
    }
 
+
+   /**
+    * The ID that uniquely identifies the channel that this transaction was received through.
+    * @since 3.31.0
+    */
+   public Originator channelId(String channelId) {
+      this.channelId = channelId;
+      return this;
+   }
+
+   @ApiModelProperty(required = false, value = "The ID that uniquely identifies the channel that this transaction was received through.")
+   @JsonProperty("channelId")
+   @Length(max = 30)
```

Comment:
The airtime API already has [ChannelProductListing](https://electrumpayments.github.io/airtime-service-interface-docs/specification/definitions/#channelproductlisting) which has a max length of 50 for the `channelName`. I don't think we need to explicitly define the name as equal to or not equal to the ID but since the `channelName` is already defined, I can imagine the `channelName` value also being used as the ID in which case I think we should make this field a little longer to match `channelName`:

```suggestion
   @Length(max = 50)
```